packages/integration-karma/test/component/LightningElement.toString/index.spec.js

```diff
@@ -3,14 +3,12 @@ import { createElement } from 'lwc';
 import Anonymous from 'x/anonymous';
 import Named from 'x/named';
 
-// TODO [#1033]: with the transformation done by the compiler, the class name is removed, producing "[object undefined]"
-xit('should rely on the constructor name', () => {
+it('should rely on the constructor name', () => {
     const elm = createElement('x-named', { is: Named });
     expect(elm.getToString()).toBe('[object MyFancyComponent]');
 });
 
-// TODO [#1033]: open issue return "[object ]" on Safari
-xit('should fallback to BaseLightningElement if constructor has no name', () => {
+it('should fallback to BaseLightningElementConstructor if constructor has no name', () => {
     const elm = createElement('x-anonymous', { is: Anonymous });
-    expect(elm.getToString()).toBe('[object BaseLightningElement]');
+    expect(elm.getToString()).toBe('[object BaseLightningElementConstructor]');
```

Comment:
Not sure what to do here, the `this` value of a component is different in compat mode so `String(this)` returns `_class`.